test/spec/modules/mass_spec.js

```diff
@@ -113,4 +117,35 @@ describe('MASS Module', function() {
     addListenerOnce();
     expect(listenerAdded).to.equal(true);
   });
+
+  it('should support custom renderers', function() {
+    init({
+      renderUrl: 'http://...',
+      custom: [
+        {
+          dealIdPattern: /abc/,
+          render: function() {}
+        }
+      ]
+    });
+
+    const renderers = updateRenderers();
+
+    expect(renderers.length).to.equal(2);
+  });
+
+  it('should match bids by deal ID with the default matcher', function() {
+    const match = useDefaultMatch(/abc/);
+
+    expect(match({dealId: 'abc'})).to.equal(true);
+    expect(match({dealId: 'xyz'})).to.equal(false);
+  });
+
+  it('should have a default renderer', function() {
+    const render = useDefaultRender('http://example.com/render.js', 'abc');
```

Comment:
Can you make this secure, and in the above place as well? Even though it isn't used,  we occasionally grep the repo for nonsecure urls to ensure secure everywhere compliance

Reply:
@patmmccann it's done